dragonfly/grammar/elements_compound.py

```diff
@@ -218,11 +222,40 @@ class Choice(Alternative):
                     "previous"      : "pgup",
                 }),
             ]
+
+        Example using a list:
+
+        .. code:: python
+
+            # Command for recognising and typing nth words, e.g.
+            # 'type third'.
+            mapping = {
+                "type <nth>": Text("%(nth)s"),
+            }
+            extras = [
+                Choice("nth", [
+                    "first",
+                    "second",
+                    "third",
+                    "fourth",
+                    "fifth",
+                    "sixth",
+                    "seventh",
+                    "eighth",
+                    "(last | ninth)",
```

Comment:
For this choice, if "last" is spoken, would the returned value be "last | nine" or "last"? I'm not 100% sure offhand, so it would probably be a good idea for the documentation to make it clear. 😃

Reply:
Ah, yes it isn't so clear. It would be "last" in that case, since the choice strings are used as compound specs. I'll update the docs to make this clearer.

We could use the strings with `Literal` instead, but that might also be confusing.

Reply:
Great! Having them used as compound specs is handy, especially since it can return only the part of the spec that was actually spoken.

Reply:
Agreed. This is also how the `Compound` element works by default. I've updated the docs. Will merge this now.